app/transformations/MemberPaymentsTransformer.scala

```diff
@@ -177,14 +179,15 @@ class MemberPaymentsTransformer @Inject()(
               )
           val same = optInitialMemberDetail.contains(refinedCurrentMember)
           if (!same) {
+//            println(s"======== i $optInitialMemberDetail")
```

Comment:
Some left over println, can be removed later on in a separate PR if that speeds things up

Reply:
removed